libgstd/Makefile.am

```diff
@@ -59,7 +59,7 @@ libgstd_@GSTD_API_VERSION@_la_SOURCES =             \
              gstd_signal_list.c                     \
              gstd_log.c
 
-libgstd_@GSTD_API_VERSION@_la_CFLAGS =                                   \
+libgstd_@GSTD_API_VERSION@_la_CFLAGS =                                     \
```

Comment:
Fix indentation

Reply:
Are you sure? I checked but I saw everything is fine

Reply:
You are right